src/nvim/buffer_defs.h

```diff
@@ -6,6 +6,8 @@
 // for FILE
 #include <stdio.h>
 
+typedef struct file_buffer buf_T; ///< Forward declaration
```

Comment:
Should this be a normal comment `//` instead of a doc comment? It doesn't talk about the purpose of the typedef.